SocialFilter/sections/specific.txt

```diff
@@ -127,8 +127,6 @@ rtvi.com##.soc
 m-cacm.acm.org##.social-links_wrapper
 cacm.acm.org##.fav_bar > a#sendByEmail
 cacm.acm.org##.fav_bar > a[original-title^="Share"]
-indirimdemarket.com#$##social-media-modal { display: none!important; }
```

Comment:
`! in 2 files`
if in Social + Annoyances=>popups, it is OK(for all removed rules)

Reply:
I mean the same rule is in a different file of Social filters.

Reply:
Should I revert, or will you?

Reply:
No, leave as is.